static/app/views/issueList/utils/useFetchIssueTags.tsx

```diff
@@ -189,16 +189,11 @@ export const useFetchIssueTags = ({
     });
 
     featureFlagTags.forEach(tag => {
```

Comment:
Optional but you've called this captured variable `tag`.  We should probably rename it to `flag`.